hepdata_validator/data_file_validator.py

```diff
@@ -121,7 +121,8 @@ def validate(self, **kwargs):
                 json_validate(data, custom_schema)
             else:
                 json_validate(data, default_data_schema)
-                if self.schema_version != '0.1.0':
+                major_schema_version = int(self.schema_version.split('.')[0])
```

Comment:
I was going to suggest wrapping this in a try/except, but actually I think checking for a valid schema version should happen when the validator is initialised (mea culpa)- I'll log a separate issue for that.

Reply:
I've logged #12 to reflect this.